Services/CO.CDP.Organisation.WebApi/Api/Organisation.cs

```diff
@@ -101,6 +101,31 @@ await useCase.Execute(organisationId)
                return operation;
            });
 
+        app.MapGet("/organisations/{organisationId}/extended",
+            [OrganisationAuthorize(
+                [AuthenticationChannel.OneLogin, AuthenticationChannel.ServiceKey],
+                [Constants.OrganisationPersonScope.Admin, Constants.OrganisationPersonScope.Editor, Constants.OrganisationPersonScope.Viewer],
+                OrganisationIdLocation.Path,
+                [Constants.PersonScope.SupportAdmin])]
+        async (Guid organisationId, IUseCase<Guid, Model.OrganisationExtended?> useCase) =>
+        await useCase.Execute(organisationId)
+                   .AndThen(organisation => organisation != null ? Results.Ok(organisation) : Results.NotFound()))
+            .Produces<Model.Organisation>(StatusCodes.Status200OK, "application/json")
+            .Produces<ProblemDetails>(StatusCodes.Status401Unauthorized)
+            .Produces<ProblemDetails>(StatusCodes.Status404NotFound)
+            .Produces<ProblemDetails>(StatusCodes.Status500InternalServerError)
+            .WithOpenApi(operation =>
+            {
+                operation.OperationId = "GetOrganisationExtended";
+                operation.Description = "Get an organisation by ID.";
+                operation.Summary = "Get an organisation by ID with additional details.";
+                operation.Responses["200"].Description = "Extended Organisation details.";
+                operation.Responses["401"].Description = "Valid authentication credentials are missing in the request.";
+                operation.Responses["404"].Description = "Organisation not found.";
+                operation.Responses["500"].Description = "Internal server error.";
+                return operation;
+            });
+
         app.MapPatch("/organisations/{organisationId}",
```

Comment:
The "extended" organisation endpoint - so I could fetch additional review data.

Maybe if we had this simple "review_status" column then we could remove this additional endpoint and just return it from the main one.